CHANGELOG.txt

```diff
@@ -51,6 +51,9 @@ All notable changes to this project starting with the 0.6.0 release will be docu
 - `site jobs` will be removed in v0.7.0 (#412)
 - `site notifications` will be removed in v0.7.0 (#412)
 
+###Removed
+- Removed --branch-create flag from `sites code`
```

Comment:
While we're at it, can we rename the filename to `CHANGELOG.md` so it gets formatted in github. :goat: